mod_config/models.py

```diff
@@ -1,6 +1,5 @@
 import json
 import os
-
```

Comment:
Please don't remove this line? :+1:

Reply:
@canihavesomecoffee Added a newline. I think the request can now be merged.